$NetBSD: patch-ae,v 1.1 2011/06/21 18:14:30 tron Exp $

Fix build under Mac OS X Snow Leopard if the Mac OS X Leopard SDK
is not installed.

--- configure.orig	2011-06-21 19:02:21.000000000 +0100
+++ configure	2011-06-21 19:04:37.000000000 +0100
@@ -105,6 +105,7 @@
 all_platforms="${all_platforms} x86-darwin8-icc"
 all_platforms="${all_platforms} x86-darwin9-gcc"
 all_platforms="${all_platforms} x86-darwin9-icc"
+all_platforms="${all_platforms} x86-darwin10-gcc"
 all_platforms="${all_platforms} x86-linux-gcc"
 all_platforms="${all_platforms} x86-linux-icc"
 all_platforms="${all_platforms} x86-solaris-gcc"
